src/app/docs/sectioned-form/sectioned-form-docs.component.html

```diff
@@ -0,0 +1,35 @@
+<sky-docs-demo-page
+  moduleName="SkySectionedFormModule"
+  moduleSourceCodePath="src/app/public/modules/sectioned-form/"
+  pageTitle="Sectioned form"
+>
+  <sky-docs-demo-page-summary>
+    The page summary displays critical information and actions for users to access quickly and frequently.
+  </sky-docs-demo-page-summary>
+
+  <sky-docs-design-guidelines>
+    <sky-docs-demo-page-section
+      heading="Usage"
+    >
+      <sky-docs-design-guideline
+        heading="Use when"
+        headingStyle="success"
+      >
+        <p class="sky-docs-text-block">
+          Use sectioned forms when users are navigating into a hierarchy and can be expected to know what they are looking for. This is common for editing pieces of a larger record and for tasks such as applying filter
+        </p>
+        <p class="sky-docs-text-block">
+          Use sectioned forms when it’s beneficial to group related but independent forms to simplify the UI on the page. For example, one “Add filters” button instead of “Filter by giving,” “Filter by status,” “Filter by constituent code,” “Filter by ratings,” etc.
```

Comment:
```suggestion
          Use sectioned forms when it’s beneficial to group related but independent forms to simplify the UI on the page. For example, use an Add filters button to open a sectioned form with multiple filter options instead of separate buttons and separate forms to access options to filter by giving, status, constituent code, ratings, or other options.
```